django_geo/__init__.py

```diff
@@ -31,14 +38,14 @@ def __ne__(self, other):
         return not (self == other)
     def __init__(self, *args, **kwargs):
         if 'lat' in kwargs and 'lng' in kwargs:
-            self.lat = Decimal(unicode(kwargs.pop('lat')))
-            self.lng = Decimal(unicode(kwargs.pop('lng')))
+            self.lat = Decimal(str(kwargs.pop('lat')))
```

Comment:
I think that in this case part of the concern might be that `kwargs['lat']` is a float, which needs to be converted to a string before being passed into `Decimal`... but at a higher level, for this PR my goal was to make the changes necessary for compatibility rather than trying to make the implementation more reasonable. (IMO this repository should not exist.)

All the changes to python code were made by running futurize (except for a couple changes to the expected django import paths).